notebooks/tutorials/cmle/sklearn_census.ipynb

```diff
@@ -0,0 +1,543 @@
+    "        lbn = LabelBinarizer()\n",
+    "        r = skb.transform(train_features)\n",
+    "        lbn.fit(r)\n",
+    "        # Create the pipeline to extract the categorical feature\n",
+    "        categorical_pipelines.append(\n",
+    "            ('categorical-{}'.format(i), Pipeline([\n",
+    "                ('SKB-{}'.format(i), skb),\n",
+    "                ('LBN-{}'.format(i), lbn)])))\n",
+    "\n",
+    "# Create pipeline to extract the numerical features\n",
+    "skb = SelectKBest(k=6)\n",
+    "# From COLUMNS use the features that are numerical\n",
+    "skb.scores_ = [1, 0, 1, 0, 1, 0, 0, 0, 0, 0, 1, 1, 1, 0]\n",
+    "categorical_pipelines.append(('numerical', skb))\n",
+    "\n",
+    "# Combine all the features using FeatureUnion\n",
+    "preprocess = FeatureUnion(categorical_pipelines)\n",
+    "\n",
+    "# Create the classifier\n",
+    "classifier = RandomForestClassifier()\n",
+    "\n",
+    "# Transform the features and fit them to the classifier\n",
+    "classifier.fit(preprocess.transform(train_features), train_labels)\n",
+    "\n",
+    "# Create the overall model as a single pipeline\n",
+    "pipeline = Pipeline([\n",
+    "    ('union', preprocess),\n",
+    "    ('classifier', classifier)\n",
+    "])\n",
+    "\n",
+    "\n",
+    "# ---------------------------------------\n",
+    "# 2. Export and save the model to GCS\n",
+    "# ---------------------------------------\n",
+    "# Export the model to a file\n",
+    "\n",
+    "#model = 'model.joblib'\n",
+    "#joblib.dump(pipeline, model)\n",
+    "\n",
+    "model = 'model.pkl'\n",
+    "\n",
+    "with open(model, 'wb') as model_file:\n",
+    "    pickle.dump(pipeline, model_file)\n",
+    "\n",
+    "# Upload the model to GCS\n",
+    "bucket = storage.Client().bucket(bucket_name)\n",
+    "blob = bucket.blob('{}/{}'.format(output_dir, model))\n",
+    "blob.upload_from_filename(model)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before you can run your trainer application with ML Engine, your code and any dependencies must be placed in a Google Cloud Storage location that your Google Cloud Platform project can access. You can find more info here"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile ./census_training/__init__.py\n",
+    "# Note that __init__.py can be an empty file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step Three: Submit Training Job\n",
+    "Next you need to submit the job for training on ML Engine. You'll use gcloud to submit the job which has the following flags:\n",
+    "\n",
+    "| Argument | Description |\n",
+    "| :--- | :----------- |\n",
+    "| job-name | A name to use for the job (mixed-case letters, numbers, and underscores only, starting with a letter). In this case: `census_training_$(date +\"%Y%m%d_%H%M%S\")` |\n",
```

Comment:
Can you add back the backticks for the argument names? I think it might also be good to include the whole thing for clarity (ex. `--job-name`) but it's up to you.